physionet-django/environment/services.py

```diff
@@ -85,14 +85,14 @@ def _create_workbench_kwargs(
         "region": region,
         "environment_type": environment_type,
         "instance_type": instance_type,
-        "dataset": _project_dataset(
+        "group_granting_data_access": _project_data_access(
```

Comment:
I'd rename it to `_project_data_group` for project and `_environment_data_group` for environment (the two functions at the top of this file)